src/main/java/carracing/domain/Cars.java

```diff
@@ -0,0 +1,54 @@
+package carracing.domain;
+
+import carracing.util.RandomUtil;
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private List<Car> cars = new ArrayList<>();
+
+    public Cars(String carNames) {
+        saveCarNames(carNames);
+    }
+
+    public List<Car> getCars() {
+        return this.cars;
+    }
+
+    private void saveCarNames(String carNames) {
+        if ("".equals(carNames.trim()))
+            throw new IllegalArgumentException("자동차 이름을 입력해주세요.");
+
+        String[] names = carNames.split(",");
+        Arrays.stream(names).forEach(name -> this.cars.add(new Car(name)));
+    }
+
+    public void play() {
+        cars.stream().forEach(car -> car.move(RandomUtil.randomNumber()));
+    }
+
+    public String getWinners() {
+        return getWinnerNames(getMaxScore());
+    }
+
+    private String getWinnerNames(int maxScore) {
+        List<Car> winners = new ArrayList<>();
+        for (Car car : this.cars) {
+            if (car.isWinner(maxScore))
+                winners.add(car);
+        }
+
+        return winners.stream().map(x -> x.getName().getName()).collect(Collectors.joining(", "));
```

Comment:
```suggestion
        return winners.stream()
                .map(Car::getName)
                .map(Name::getName)
                .collect(Collectors.joining(", "));
```
이렇게 Stream 관련 메소드는 한 줄에 하나씩만 작성해주세요!